### DISCUSSION TITLE
rhonabwy for windows

### DISCUSSION THREAD

**Discussion by ekhanad** [babelouest/rhonabwy#5] [Q&A]
I wonder if rhonabwy is available for windows? Or is it possible to compile on windows?

**Comment by babelouest** [comment]
Hello,

That's possible, considering its dependencies are available on Windows AFAICT, you should be able to build and run rhonabwy for Windows.

At least with yacp, maybe with visual studio, see here for details on running ulfius with visual studio: https://github.com/babelouest/ulfius/issues/111
